main/src/com/google/refine/ProjectMetadata.java

```diff
@@ -263,4 +329,110 @@ public void setCustomMetadata(String key, Serializable value) {
         }
         updateModified();
     }
+    
+    public JSONArray getImportOptionMetaData() {
+        return _importOptionMetaData;
+    }
+    
+    public void setImportOptionMetaData(JSONArray jsonArray) {
+        _importOptionMetaData = jsonArray;
+        updateModified();
+    }
+    
+    public void appendImportOptionMetaData(JSONObject obj) {
+        _importOptionMetaData.put(obj);
+        updateModified();
+    }
+
+    
+    public String getCreator() {
+        return _creator;
+    }
+
+    
+    public void setCreator(String creator) {
+        this._creator = creator;
+        updateModified();
+    }
+
+    
+    public String getContributors() {
+        return _contributors;
+    }
+
+    
+    public void setContributors(String contributors) {
+        this._contributors = contributors;
+        updateModified();
+    }
+
+    
+    public String getSubject() {
+        return _subject;
+    }
+
+    
+    public void setSubject(String subject) {
+        this._subject = subject;
+        updateModified();
+    }
+
+    
+    public String getDescription() {
+        return _description;
+    }
+
+    
+    public void setDescription(String description) {
+        this._description = description;
+        updateModified();
+    }
+
+    
+    public int getRowNumber() {
+        return _rowNumber;
+    }
```

Comment:
What is this `rowNumber` metadata? If that counts the number of rows in the project, shouldn't it rather be `rowCount`? `rowNumber` reads like a row index…